apps/common/lib/lexical/ast.ex

```diff
@@ -15,10 +65,13 @@ defmodule Lexical.Ast do
 
   @type t :: any()
 
+  @type cursor_context :: any()
+  @type surround_context :: any()
```

Comment:
Note to self: actually write these out. And possibly move the fragment api to `Lexical.Ast.Fragment`

Reply:
I'm actually not sure how/whether I want to handle this. It feels a little silly to fully duplicate the return types from `Code.Fragment` (especially since they define the return fully in the function spec, not as separate types). I'm sort of more inclined to just document these as representing the return values from those functions. 

```elixir
@typedoc "Return value from `Code.Fragment.cursor_context/2`"
@type cursor_context :: any()
```

Reply:
@scohen @scottming WDYT?

Reply:
I don't have a strong opinion, though I shy away from `any()` by default. I think the typedoc clears it up.

Reply:
I'm on the same page as steve, I think typedoc is clear enough here.